pkg/registry/tags.go

```diff
@@ -0,0 +1,87 @@
+package registry
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/giantswarm/microerror"
+)
+
```

Comment:
I feel like we should have two separate implementations for something like `TagLister` interface and wire a correct one instead of switching on `r.kind` all the time.

Reply:
I fully agree with you here. I'll try to come up with a valid implementation